src/test/java/com/cloudant/tests/IndexTests.java

```diff
@@ -72,6 +70,16 @@ public void indexTestAll() {
         db.createIndex("Movie_year", "Movie_year", null,
                 new IndexField[]{new IndexField("Movie_year", SortOrder.asc)});
 
+        // selectorJson as a proper json object

```

Comment:
This was copied over with the succeeding lines.  It's referencing the `selector` variable, but this may at one point have been called `selectorJson`.  I can either update this comment or remove it.